modules/widgets/social-icons.php

```diff
@@ -512,6 +512,106 @@ public function get_supported_icons() {
 				'icon'  => 'feed',
 				'label' => __( 'RSS Feed', 'jetpack' ),
 			),
+			array(
+				'url'   => '/feeds/',
+				'icon'  => 'feed',
+				'label' => __( 'RSS Feed', 'jetpack' ),
+			),
+			array(
+				'url'   => 'format=RSS',
+				'icon'  => 'feed',
+				'label' => __( 'RSS Feed', 'jetpack' ),
+			),
+			array(
+				'url'   => '/rss', // Used by Tumblr
+				'icon'  => 'feed',
+				'label' => __( 'RSS Feed', 'jetpack' ),
+			),
+            array(
+				'url'   => '/.rss', // Yep, Reddit uses this
```

Comment:
I think those types of comments are helpful, it helps us see at a glance what sites we support. Do you think you could add some more for the other formats in there?

Reply:
Sure, I'll see what other ones I can provide comments for. The ones I commented were very specific sources, the others aren't necessarily from one particular source of RSS but I'll see what I can add.

Reply:
I've added some comments for most of the RSS formats now, a few of them are unknown / generic ones I've come across so not specifically with any particular CMS / platform.